app/presenters/concerns/qualification_api_data.rb

```diff
@@ -151,11 +151,17 @@ def structured_gcse_to_hashes(gcse)
 private
 
   def completing_qualification(qualification)
-    return {} unless include_completing_qualification? && qualification.gcse?
+    return {} unless include_completing_qualification?
+
+    if qualification.gcse?
+      return {
+        currently_completing_qualification: qualification[:currently_completing_qualification],
+        missing_explanation: qualification[:missing_explanation],
```

Comment:
It looks like we were including these fields for non-gcse qualifications in the past. are they always nil?

Reply:
Have I messed up my if's and unless' 🤦‍♂️

Reply:
We were only including these fields if the qualification was a gcse. We are now including the `other_uk_qualification_type` for all qualifications. 
I've waled through how I refactored this and came to the same code :😅